src/lib/rules/meta-viewport/meta-viewport.ts

```diff
@@ -0,0 +1,208 @@
+/**
+ * @fileoverview Check if the viewport meta tag is specified in the
+ * `<head>` with the proper value.
+ */
+
+import { parseMetaViewPortContent } from 'metaviewport-parser';
+
+import { isHTMLDocument, normalizeString } from '../../utils/misc';
+import { IAsyncHTMLDocument, IAsyncHTMLElement, ITraverseEnd } from '../../types'; // eslint-disable-line no-unused-vars
+import { IRule, IRuleBuilder } from '../../types'; // eslint-disable-line no-unused-vars
+
+import { RuleContext } from '../../rule-context'; // eslint-disable-line no-unused-vars
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+const rule: IRuleBuilder = {
+    create(context: RuleContext): IRule {
+
+        // This function exists because not all connector (e.g.: jsdom)
+        // support matching attribute values case-insensitively.
+        //
+        // https://www.w3.org/TR/selectors4/#attribute-case
+
+        const getViewportMetaTags = (elements: Array<IAsyncHTMLElement>): Array<IAsyncHTMLElement> => {
+            return elements.filter((element) => {
+                return (element.getAttribute('name') !== null && normalizeString(element.getAttribute('name')) === 'viewport');
+            });
+        };
+
+        const listIncludesBrowsersWithOrientationChangeBug = (browsersList: string[]): boolean => {
+
+            // Old versions of Safari for iOS require `initial-scale=1`
+            // in order to react on orientation change.
+            //
+            //  * https://www.quirksmode.org/blog/archives/2013/10/initialscale1_m.html
+            //  * https://www.quirksmode.org/blog/archives/2013/10/more_about_scal.html
+            //  * https://www.quirksmode.org/mobile/metaviewport/quickdevicewidth.html
+            //
+            // As of Safari for iOS 9+, that is no longer the case.
+            //
+            //  * https://twitter.com/ppk/status/829329905942986752
+
+            // If no browsers are targeted, assume Safari for iOS < 9
+            // are also supported.
+
+            if (browsersList.length === 0) {
+                return true;
+            }
+
+            return browsersList.some((e) => {
```

Comment:
Call me picky but, `e` sound like an error or event to me. Maybe you can call it `browser` or something similar.

Reply:
>Note: I generate the some and similar things automatically in my editor, and e comes from element, so if you see it in the future, that is why.

ahhh hehehe.

>Let's go with `browserVersion`.

Perfect!